### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Nikkarchinskiy**

### CODE REVIEW

File: homeworks/nikita.karchinskiy_Nikkarchinskiy/5-rxjs/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Document</title>
+    </head>
+    <body>
+        <header class="header" data-header>
+            <div class="header-wrapper-l">
+                <a class="header-logo" href="#">Logo</a>
+                <nav class="header-nav">
+                    <menu class="menu-nav">
```
Comment (DrSmile444) [inline homeworks/nikita.karchinskiy_Nikkarchinskiy/5-rxjs/index.html:15]: You don't need the `menu` tag here. It's better to remove it:
w3schools.com/tags/tag_nav.asp

**Review by DrSmile444 — CHANGES_REQUESTED** [review]
There's no JS. Nothing to review

**Review by DrSmile444 — CHANGES_REQUESTED** [review]
There's no Github Pages link still.

File: homeworks/nikita.karchinskiy_Nikkarchinskiy/5-rxjs/rx.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const menuBurger$ = fromEvent(iconMenuEl, 'click');
+
+scrolling$.pipe(
+    map(() => window.scrollY),
+    throttleTime(190),
+    pairwise(),
+    filter((pixel) => (pixel[0] - pixel[1]) > 50 || (pixel[1] - pixel[0]) > 50),
+    map((pixel) => pixel[1] < pixel[0]),
+).subscribe((e) => {
+    if (e) {
```
Comment (DrSmile444) [inline homeworks/nikita.karchinskiy_Nikkarchinskiy/5-rxjs/rx.js:23]: What is e? I don't understand. Find a better name for this variable, please

**Comment by Nikkarchinskiy** [comment]
Я сделал но линтер уже не пропускает название ветки, как я понял

**Comment by Nikkarchinskiy** [comment]
https://nikkarchinskiy.github.io/frontend-course-2021/homeworks/nikita.karchinskiy_Nikkarchinskiy/5-rxjs/index.html

File: package.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
         "eslint-config-airbnb-base": "^14.2.1",
         "eslint-plugin-import": "^2.24.2",
         "eslint-plugin-no-jquery": "^2.6.0",
+        "rxjs": "^7.4.0",
```
Comment (DrSmile444) [inline package.json:22]: I see here an extra rxjs package

**Review by DrSmile444 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**